neofsid/neofsid_contract.go

```diff
@@ -33,6 +33,7 @@ func _deploy(data interface{}, isUpdate bool) {
 	ctx := storage.GetContext()
 
 	if isUpdate {
+		storage.Delete(ctx, common.LegacyOwnerKey)
 		it := storage.Find(ctx, []byte{}, storage.None)
```

Comment:
Can you drop the code below in `isUpdate` branch? It was used for `v0.13.x` update and is not related anymore.